### PR TITLE
Update saxe2013exact_1.md

### PR DESCRIPTION
**Pull request by liyanc**

### CODE REVIEW

File: _summaries/saxe2013exact_1.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+Therefore, the result is a set of vectors that are either pairwise orthogonal or colinear.
+The authors further examine the invariance manifold of weight vectors and find that their modes in the spectra evolve independently and staying orthogonal.
+With similar but limited noisy initial condition, the authors find that the learning speed follows a sigmoid function as described in
+$$
+u_{f}(t)=\frac{s e^{2 s t / \tau}}{e^{2 s t / \tau}-1+s / u_{0}}
+$$
+, which coincides with the experiment well.
+
+  Then the authors generalize to a multi-layer linear network case assuming all weights are orthogonal.
+The differential equation can be therefore written as $$\tau \frac{d}{d t} u=N_{l} u^{2}(s-u)$$, which imples that the learning speed converges to constant when the number of layers goes to infinity.
```
Comment (nilesh2797) [inline _summaries/saxe2013exact_1.md:39]: The equation doesn't imply the conclusion

File: _summaries/saxe2013exact_1.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
...
 * How well does the paper perform?
```
Comment (nilesh2797) [inline _summaries/saxe2013exact_1.md:40]: Discussion of orthogonal random initialization and its scaled-up version for nonlinear deep nets is missing

File: _summaries/saxe2013exact_1.md [hunk 3]
```diff
@@ -1,19 +1,51 @@
...
+  The author proposed a pretraining scheme accordingly by first auto-encoding the input data and switching to the actual training.
```
Comment (nilesh2797) [inline _summaries/saxe2013exact_1.md:41]: *correction*: proposed -> analyzed

File: _summaries/saxe2013exact_1.md [hunk 4]
```diff
@@ -1,19 +1,51 @@
...
+They perform experiments to validate the claim as well.
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Solving linear networks analytically as diffeqs demonstrates properties (symmetries, invariants) which have implications on the learning dynamics of them.
+* With such invariants, there is a manifold where the learning trajectory is optimal
+* Initializing the weights on the manifold gives rise to fast and converging learning dynamics, which is validated by experiments
+
```
Comment (nilesh2797) [inline _summaries/saxe2013exact_1.md:51]: These TLDR points miss some of the core claims of the paper like 1) deep linear neural nets with right initialization are only constant amount slower to train than shallower ones 2) random orthogonal initializations can outperform random gaussian initialization (glorot initialization)

**Review by zhaoyue-zephyrus — APPROVED** [review]
